src/main/java/com/epam/auto/test/level2/pages/GoogleCloudPricingCalculatorPage.java

```diff
@@ -0,0 +1,98 @@
+        WebElement choseMachineClass = waitWebElementAndClick("(//md-option[@value='regular'])[last()]");
+
+        WebElement machineTypes = waitWebElementAndClick("//*[text()='Machine type']/following::md-select[1]");
+        WebElement choseMachineType = waitWebElementAndClick("//md-option[@value='CP-COMPUTEENGINE-VMIMAGE-N1-STANDARD-8']");
+
+        WebElement addGPUs = waitWebElementAndClick("//md-checkbox[@ng-model='listingCtrl.computeServer.addGPUs']");
+
+        WebElement numbersOfGPUs = waitWebElementAndClick("//*[text()='Number of GPUs']/following::md-select[1]");
+        WebElement choseNumberOfGPUs =
+                waitWebElementAndClick("//md-option[@ng-repeat='item in listingCtrl.supportedGpuNumbers[listingCtrl.computeServer.gpuType]' and @value='1']");
+
+        WebElement typesGPU = waitWebElementAndClick("//*[text()='GPU type']/following::md-select[1]");
+        WebElement choseTypeGPU = waitWebElementAndClick("//md-option[@value='NVIDIA_TESLA_V100']");
+
+        WebElement localSSDs = waitWebElementAndClick("//*[text()='Local SSD']/following::md-select[1]");
+        WebElement choseLocalSSD = waitWebElementAndClick("//md-option[@ng-repeat='item in listingCtrl.supportedSsd' and @value='2']");
+
+        WebElement datacenterLocations = waitWebElementAndClick("//*[@ng-model='listingCtrl.computeServer.location']");
+        WebElement choseDatacenterLocation = waitWebElementAndClick("(//md-option[@value='europe-west3'])[last()]");
+
+        WebElement committedUsages = waitWebElementAndClick("//*[@ng-model='listingCtrl.computeServer.cud']");
+        WebElement selectCommittedUsage = waitWebElementAndClick("(//div[text()='1 Year'])[last()]");
+
+        WebElement addToEstimate = waitWebElementAndClick("//button[@ng-click='listingCtrl.addComputeServer(ComputeEngineForm);']");
+        return this;
+    }
+
+    public GoogleCloudPricingCalculatorPage sendEmail(String email) {
+        WebElement emailEstimate = waitWebElementAndClick("//button[@id='email_quote']");
+        inputEmail.sendKeys(email);
+        WebElement sendEmail = waitWebElementAndClick("//button[@ng-click='emailQuote.emailQuote(true); emailQuote.$mdDialog.hide()']");
+        return this;
+    }
+
+    public boolean checkEstimate() {
+        boolean check = false;
+        boolean vmClass = checkValue("//*[@id='compute']//md-list-item[2]", "regular");
+        boolean instanceType = checkValue("//*[@id='compute']//md-list-item[3]", "n1-standard-8");
+        boolean region = checkValue("//*[@id='compute']//md-list-item[4]", "Frankfurt");
+        boolean localSSD = checkValue("//*[@id='compute']//md-list-item[5]", "2x375 GB");
+        boolean commitmentTerm = checkValue("//*[@id='compute']//md-list-item[6]", "1 Year");
+        if (vmClass && instanceType && region && localSSD && commitmentTerm) {
+            check = true;
+        }
+        return check;
```

Comment:
just return vmClass && instanceType && region && localSSD && commitmentTerm;
boolean check is unnessary variable